src/Api.Contracts/Requests/AddProductRequest.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Api.Contracts.Requests;
+﻿namespace CleanCompanyName.DDDMicroservice.Api.Contracts.Requests;
```

Comment:
Is it tested? Doesn't it need to be:
```suggestion
﻿namespace $CleanCompanyName$.$DDDMicroservice$.Api.Contracts.Requests;
```
?

Example from standard Visual Studio templates I've found:
![image](https://user-images.githubusercontent.com/19998579/165158297-e5507faf-87d0-40c8-bd54-4640b46f34ca.png)

Reply:
It's tested and it works.
I guess the $$ way is when you need to evaluate the value.
In this case, it's only a replacement which will be safe as far as we don't use the same word for any other thing besides the namespaces. :-)

Reply:
Doesn't sound safe 😨